services/app-web/src/pages/SubmissionRevisionSummary/SubmissionRevisionSummaryV2.tsx

```diff
@@ -54,7 +54,7 @@ export const SubmissionRevisionSummaryV2 = (): React.ReactElement => {
 
     const name =
         contract &&
-        contract?.packageSubmissions.length < Number(revisionVersion)
+        contract?.packageSubmissions.length > Number(revisionVersion)
```

Comment:
We might want to revisit this log later. Why on earth would we be rendering this page if theres not enough submissions lol. I would expect us to show an error or something here right - not just show an empty name string

Reply:
we do show an error below, but bc the name is rendered with the updateHeading effect we have to repeat the check here, I think